freqtrade/templates/strategy_analysis_example.ipynb

```diff
@@ -48,12 +49,12 @@
     "# Load data using values set above\n",
     "from freqtrade.data.history import load_pair_history\n",
     "\n",
-    "candles = load_pair_history(datadir=data_location,\n",
-    "                            timeframe=timeframe,\n",
+    "candles = load_pair_history(datadir=config[\"datadir\"],\n",
```

Comment:
exchange should be selected somehow...
this does not find the datafiles (in the top `datadir`)